reference/user_defined_functions.rst

```diff
@@ -202,10 +202,76 @@ Example
 Metadata / Configuration Information
 ------------------------------------------------------------------------
 
+.. _master_add_node:
+
+master_add_node
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+The master_add_node() function registers a new node addition in the cluster in
+the Citus metadata table pg_dist_node.
+
+Arguments
+************************
+
+**node_name:** DNS name of the new node to be added.
+
+**node_port:** The port on which PostgreSQL is listening on the worker node.
+
+Return Value
+******************************
+
+A tuple which represents a row from pg_dist_node table. See the metadata table
+reference section for more information on the table.
+
+
+Example
+***********************
+
+::
+
+    select master_add_node('new-node', 12345);
+            master_add_node         
+    --------------------------------
+     (5,5,new-node,12345,default,f)
+    (1 row)
+
+master_remove_node
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+The master_remove_node() function removes the specified node from the
+pg_dist_node metadata table. This function will error out if there are existing
+shard placements on this node in pg_dist_shard_placement. Thus, before using
```

Comment:
(Minor) Maybe make it `This function will error out if there are existing shard placements on this node (by referring to pg_dist_shard_placement.)` or something similar.

I like it as this separates what its trying to do from how its doing it. But, I don't have a strong opinion on this, so feel free to ignore if you like the current version better :).